### PR TITLE
Bump to Kogito 0.15.0

### PR DESCRIPTION
**Pull request by evacchi**
I am actually bumping to 1.0.0-SNAPSHOT (master nightly). We should release 0.15.0 on Friday

### CODE REVIEW

**Comment by evacchi** [comment]
alright, so I suppose snapshots won't work :sweat_smile:

File: pom.xml [hunk 1]
```diff
@@ -82,6 +82,19 @@
         <module>integration-tests</module>
     </modules>
 
+    <repositories>
+        <repository>
+            <id>jboss-public-repository-group</id>
+            <name>JBoss Public Repository</name>
+            <snapshots>
+              <enabled>false</enabled>
```
Comment (gsmet) [inline pom.xml:90]: Maybe you should enable the snapshots if you want them to work :].

Comment (evacchi) [reply pom.xml:90]: 😅😅😅😅

**Comment by evacchi** [comment]
ok kogito tests ran successfully. I am not sure what's the deal with optaplanner though -- but it says something about the snapshot package, so maybe that's just about the snapshot being used there.

**Comment by gsmet** [comment]
I think it probably got this artifact from the JBoss repository and you forcefully disabled the releases there.

**Comment by evacchi** [comment]
I disabled it because the previous run took >60 minutes

**Pull request merged**